docs/ref/applications.txt

```diff
@@ -346,7 +347,7 @@ application registry.
     * When invoking a management command.
 
     It must be called explicitly in other cases, for instance in plain Python
-    scripts.
+    scripts. ``set_script`` is only useful outside of requests.
```

Comment:
I would say it's only useful if you are writing your own request/response handling since if you are outside of requests, you probably want the default of True, right?